README.md

```diff
@@ -20,6 +20,11 @@ Fix-up for gcc 4.8.3
 > chmod +x zzcpy2gccfixed.sh
 > ./zzcpy2gccfixed.sh
 ```
+Fixup for gcc 6.3 from IBM yum repo
+cd storage/ibmdb2i/special
+cp /usr/include/unistd.h
+cp /QIBM/include/qmyse.h
```

Comment:
@abmusse , do you know why these steps aren't required for your builds you've done?

Reply:
Looks there is `qmyse.h` in `special/include`  already 

https://github.com/zendtech/db2iengine/blob/master/special/include/qmyse.h

We prepend the `special/include` to our include directories here:

https://github.com/zendtech/db2iengine/blob/master/CMakeLists.txt#L3

https://cmake.org/cmake/help/v3.7/command/include_directories.html#command:include_directories


I'm not sure why `/usr/include/unistd.h` is needed though. @davidl-zend  Can you chime in on this?